core/src/main/java/bisq/core/dao/state/model/blockchain/TxOutput.java

```diff
@@ -36,7 +37,8 @@
  * Gets persisted.
  */
 @Immutable
-@Data
+@Getter
+@Setter
```

Comment:
There is no write access and its Immutable. The @Data was wrong and should have been a @Value annotation.
Can you remove the @Setter?

Reply:
Setter removed in commit c2c1ac2087a8862dbf02007adf299aba460f92f5.